p5js/LSTM_Text/sketch.js

```diff
@@ -10,21 +10,18 @@ This uses a pre-trained model on a corpus of Virginia Woolf
 For more models see: https://github.com/ml5js/ml5-data-and-training/tree/master/models/lstm
 === */
 
-// Create the LSTM Generator passing it the model directory
-const lstm = ml5.LSTMGenerator('models/woolf/', modelReady);
-
+let lstm;
 let textInput;
 let lengthSlider;
 let tempSlider;
 let button;
 
-function modelReady() {
-  select('#status').html('Model Loaded');
-}
-
 function setup() {
   noCanvas();
 
+  // Create the LSTM Generator passing it the model directory
+  lstm = ml5.LSTMGenerator('./models/woolf/', modelReady);
```

Comment:
This is for later, but it's probably preferable to use `preload()` for the basic LSTM p5 example (and maybe image classification). This is simpler for total beginners.